src/RadialAreaChart/RadialAreaChart.story.tsx

```diff
@@ -100,3 +100,65 @@ export const MultiSeries = () => (
     axis={<RadialAxis type="category" />}
   />
 );
+
+export const SemiCircle = () => (
+  <RadialAreaChart
+    height={500}
+    width={500}
+    data={medDateData}
+    innerRadius={0.1}
+    series={
+      <RadialAreaSeries
+        colorScheme="cybertron"
+        animated
+        interpolation="smooth"
+        area={<RadialArea gradient={<RadialGradient />} />}
+      />
+    }
+    axis={
+      <RadialAxis
+        arcs={<RadialAxisArcSeries count={10} />}
+        ticks={
+          <RadialAxisTickSeries
+            count={5}
+            tick={
+              <RadialAxisTick
+                line={<RadialAxisTickLine position="inside" />}
+                label={<RadialAxisTickLabel autoRotate />}
+              />
+            }
+          />
+        }
+      />
+    }
+    isSemiCircle={true}
+  />
+);
+
+export const SemiCircleMultiSeries = () => (
+  <RadialAreaChart
+    data={multiCategory}
+    height={500}
+    width={500}
+    series={
+      <RadialAreaSeries
+        type="grouped"
+        interpolation="linear"
+        area={
+          <RadialArea
+            gradient={
+              <RadialGradient
+                stops={[
+                  <GradientStop offset="0%" stopOpacity={0.1} key="start" />,
+                  <GradientStop offset="80%" stopOpacity={0.3} key="stop" />
+                ]}
+              />
+            }
+          />
+        }
+      />
+    }
+    axis={<RadialAxis type="category" />}
+    isSemiCircle={true}
```

Comment:
What if we did 

```
  /**
   * Start angle for the first value.
   */
  startAngle?: number;

  /**
   * End angle for the last value.
   */
  endAngle?: number;
```

which makes it more flexible and matches other APIs in the app.

Reply:
That would require changes in other parts wherever we have used the scales and transformations but the major challenge would  be in drawing a path for the [axis](https://github.com/reaviz/reaviz/pull/142/files#diff-1023bf738c818131abcc1515e0edb35829044401dabe4c792d6ae97a944c423aR59-R63)

Reply:
I defiantly want to make it where we can set the start/end angle. That will make it way more flexible and allow us to use other angles in other projects.

Reply:
Closing this PR. Opened https://github.com/reaviz/reaviz/pull/144 with the changes for start/end angles.